<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI</class>
 <widget class="QMainWindow" name="GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1231</width>
    <height>787</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1231</width>
    <height>787</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1231</width>
    <height>787</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Hypercube</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Auwahl">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>241</width>
      <height>761</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Abspielen">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <attribute name="title">
      <string>Abspielen</string>
     </attribute>
     <widget class="QGroupBox" name="Geschwindigkeit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>106</y>
        <width>221</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Geschwindikeitseinstellungen</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>191</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Multiplikator">
          <property name="text">
           <string>1.0</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>36</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Multiplikator der Framelänge</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Multiplex">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>221</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Multiplex Einstellungen</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>191</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="MultiplexSpeed">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>ms</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="ubernehmen">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>630</y>
        <width>221</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Übernehmen</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="Warteschlange">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>221</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>Warteschalnge</string>
      </property>
      <widget class="QListWidget" name="ListSchlange">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>27</y>
         <width>198</width>
         <height>131</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>202</width>
         <height>52</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="plusWarte">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="minusWarte">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="obenWarte">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>u</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="untenWarte">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>d</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="liveubertragen">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>229</y>
        <width>221</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Live Übertragung einschalten</string>
      </property>
      <widget class="QCheckBox" name="live">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>34</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Live Übertragung</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="OnStage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>302</y>
        <width>221</width>
        <height>95</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Wird abgespielt</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>201</width>
         <height>25</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>hello.cube</string>
        </property>
       </item>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>63</y>
         <width>150</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>172</x>
         <y>63</y>
         <width>40</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Erstellen">
     <attribute name="title">
      <string>Erstellen</string>
     </attribute>
     <widget class="QGroupBox" name="frames">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>221</width>
        <height>651</height>
       </rect>
      </property>
      <property name="title">
       <string>FrameBox</string>
      </property>
      <widget class="QListWidget" name="FrameBox">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>221</width>
         <height>631</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="plusErstelle">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>671</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="minusErstelle">
      <property name="geometry">
       <rect>
        <x>67</x>
        <y>671</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="obenErstelle">
      <property name="geometry">
       <rect>
        <x>123</x>
        <y>671</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="text">
       <string>u</string>
      </property>
     </widget>
     <widget class="QPushButton" name="untenErtelle">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>671</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>55</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="text">
       <string>d</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="OpenGL" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>23</y>
      <width>970</width>
      <height>700</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
   </widget>
   <widget class="QSlider" name="rotation">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>733</y>
      <width>970</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-180</number>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1231</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionOffnen"/>
    <addaction name="actionSpeichern"/>
    <addaction name="actionSpeichernUnter"/>
    <addaction name="actionExportieren"/>
    <addaction name="actionBeenden"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menubertragen">
    <property name="title">
     <string>Einstellungen</string>
    </property>
    <addaction name="actionCOM_PortEinstellungen"/>
    <addaction name="actionToggle_Light_darkmode"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menubertragen"/>
  </widget>
  <action name="actionOffnen">
   <property name="text">
    <string>Öffnen</string>
   </property>
  </action>
  <action name="actionSpeichern">
   <property name="text">
    <string>Speichern</string>
   </property>
  </action>
  <action name="actionSpeichernUnter">
   <property name="text">
    <string>Speichern unter</string>
   </property>
  </action>
  <action name="actionExportieren">
   <property name="text">
    <string>Exportieren</string>
   </property>
  </action>
  <action name="actionNeu">
   <property name="text">
    <string>Neu</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="actionCOM_PortEinstellungen">
   <property name="text">
    <string>COM Port Einstellungen</string>
   </property>
  </action>
  <action name="actionToggle_Light_darkmode">
   <property name="text">
    <string>Toggle Light- / Darkmode</string>
   </property>
  </action>
  <action name="actionLife_bertragung">
   <property name="text">
    <string>Life Übertragung</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OpenGL</class>
   <extends>QWidget</extends>
   <header location="global">opengl.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
